archive/archiver/src/main/scala/uk/ac/wellcome/platform/archiver/messaging/MessageStream.scala

```diff
@@ -49,13 +49,20 @@ class MessageStream[T, R] @Inject()(actorSystem: ActorSystem,
       ActorMaterializerSettings(system)
         .withSupervisionStrategy(decider(metricName, Supervision.Resume)))
 
-    val typeConversion = Flow[Message].map(m => fromJson[T](m.getBody).get)
+    val typeConversion = Flow[Message].map( m => {
```

Comment:
Can we not use the `log` function from akka streams here?

Reply:
The Akka log looks a bit all or nothing to me, I wanted to log the incoming msg at `info` and leave everything else at `debug`.